scripts/build_pkgdown.R

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env Rscript
+
+# Build pkgdown documentation for PEcAn packages
+library(pkgdown)
+
+args <- commandArgs(trailingOnly = TRUE)
+if (length(args) == 0) {
+  stop("No package names provided. Please pass package names as arguments.")
+}
+
+packages <- args
+
+if (requireNamespace("PEcAn.logger", quietly = TRUE)) {
+  logger <- PEcAn.logger::logger.info
+} else {
+  logger <- function(...) {
+    message(paste(...)) 
+  }
+}
+
+logger(paste("Building pkgdown docs for:", paste(packages, collapse = ", ")))
```

Comment:
Nit: logger function will handle the outermost paste for us (which is one of my favorite features, so I like to use it 🤓)
```suggestion
logger("Building pkgdown docs for:", paste(packages, collapse = ", "))
```